agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -607,26 +635,48 @@ async def get_user_object(user_uid: str) -> UserDB:
 
```

Comment:
Can we add some comment explaining what is the reason behind this function. who calls it

Reply:
@mmabrouk 

Alright. 
It's called by almost every function in db_manager to get the user object to use in query or calling other functions.